scheduler/project.clj

```diff
@@ -155,7 +156,7 @@
                                  org.slf4j/slf4j-nop]]
                    ; Similarly, one could use an older version of the
                    ; mesomatic library in environments that require it
-                   [twosigma/mesomatic "1.0.1-2018.08.28-df2c2ca"]]}
+                   [pschorf/mesomatic "1.5.0-r4"]]}
```

Comment:
Should we push your code to `twosigma/mesomatic`?

Reply:
I was trying to develop my personal brand a bit.

Done.